README.md

```diff
@@ -107,11 +107,11 @@ When filtered, the colors represent the following:
 
 ## Example Code
 
-The most current example for using MoveIt! Grasps is the `grasp_pipeline_demo` which can be found [here](https://github.com/PickNikRobotics//moveit_grasps/kinetic-devel/src/grasp_pipeline_demo.cpp).
+The most current example for using MoveIt! Grasps is the `grasp_pipeline_demo` which can be found [here](https://github.com/ros-planning/moveit_grasps/melodic-devel/src/grasp_pipeline_demo.cpp).
 
 There are other example implementations:
 
- - [baxter_pick_place](https://github.com/davetcoleman/baxter_cpp/tree/kinetic-devel/baxter_pick_place)
+ - [baxter_pick_place](https://github.com/davetcoleman/baxter_cpp/tree/melodic-devel/baxter_pick_place)

```

Comment:
broken link. Please revert. Also, this isn't built in that repo so I'm hesitant to include it in this repo. Maybe just remove

Reply:
same. I could replace config and demo instructions by a link to the not-yet merged tutorial